harness/determined/common/storage/util.py

```diff
@@ -0,0 +1,46 @@
+import urllib
+
+from determined.common import storage
+
+
+def from_string(shortcut) -> storage.base.StorageManager:
```

Comment:
I know that the `storage` module is so old that it doesn't follow our [code layout rfc](https://github.com/rb-determined-ai/pedl/blob/9bd2168ec23fbba866194f3d7d3cfc243d267100/python_code_layout.md), so it's not obvious that this is necessary, but would you mind omitting the submodules from `storage.*`?  So use `storage.StorageManager`, not `storage.base.StorageManager`.  And similarly use `storage.GCSStorageManager`, not `storage.gcs.GCSStorageManager`.